docs/configuration/features/text_features.md

```diff
@@ -281,6 +288,32 @@ Parameters:
 {{ render_fields(schema_class_to_fields(hf_encoder, exclude=["type"])) }}
 {% endfor %}
 
+## LLM encoders
+
+``` mermaid
+graph LR
+  A["12\n7\n43\n65\n23\n4\n1"] --> B["Pretrained\n LLM"];
+  B --> C["Last\n Hidden\n State"];
+  C --> ...;
+```
+
+{ data-search-exclude }
+
+The LLM encoder processed text with a pretrained LLM (ex. `llama-2-7b`) passes the last hidden state of the LLM forward to the combiner. Like the LLM model type, adapter-based fine-tuning and quantization can be configured, and any combiner or decoder parameters will be bundled with the adapter weights.
```

Comment:
typo:

```
The LLM encoder processes text with a pretrained LLM (ex. `llama-2-7b`) and passes the last hidden state of the LLM forward to the combiner.
```

I'd also add a reference to docs for the `LLM model type` inline.

Reply:
Fixed